lib/web_ui/lib/src/engine/text/measurement.dart

```diff
@@ -690,7 +690,7 @@ double _measureSubstring(
   final String sub =
       start == 0 && end == text.length ? text : text.substring(start, end);
   final double width =
-      _canvasContext.measureText(sub).width + letterSpacing * sub.length as double;
+      _canvasContext.measureText(sub).width! + letterSpacing * sub.length as double;
```

Comment:
Surprised that measureText can return null.

Reply:
We casted a pretty wide net for attributes to make sure we don't accidentally type something as non-nullable and it ends up returning null. It's likely there are some getters that are non-nullable but will most likely never return null. For this case, `width` is tagged as experimental in the MDN, which is why we default to nullable.